packages/terra-toggle-button/tests/wdio/toggle-button-spec.js

```diff
@@ -0,0 +1,173 @@
+/* global before, browser, Terra */
+
+describe('Toggle Button', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+
+  describe('Default', () => {
+    before(() => browser.url('/#/raw/tests/terra-toggle-button/toggle-button/default-toggle-button'));
+
+    Terra.should.matchScreenshot('closed', { selector: '.button' });
+    Terra.should.beAccessible();
+
+
+    it('should open the button', () => {
+      browser.click('button');
+
+      // Ensures the on hover button styling is not displayed
+      browser.click('p*=Lorem');
+    });
+
+    Terra.should.matchScreenshot('opened', { selector: '.button' });
+    Terra.should.beAccessible();
+  });
+
+  describe('Icon Only', () => {
+    before(() => browser.url('/#/raw/tests/terra-toggle-button/toggle-button/icon-only'));
+
+    it('should open the button', () => {
+      browser.click('button');
+
+      // Ensures the on hover button styling is not displayed
+      browser.click('p*=Lorem');
+    });
+
+    Terra.should.matchScreenshot('opened', { selector: '.button' });
+    Terra.should.beAccessible();
+  });
+
+  describe('Custom Icon', () => {
+    before(() => browser.url('/#/raw/tests/terra-toggle-button/toggle-button/custom-icon'));
+
+    it('should open the button', () => {
+      browser.click('button');
+
+      // Ensures the on hover button styling is not displayed
+      browser.click('p*=Lorem');
+    });
+
+    Terra.should.matchScreenshot('opened', { selector: '.button' });
```

Comment:
Do we need to add a selector to these example?

Reply:
It will make the screenshots smaller typically because it only captures the selector area

Reply:
I was just curious because by default terra-core has a global selector defined what should keep the screenshots small